### ISSUE TITLE
How are people generating the JSON schema?

### ISSUE THREAD

**Issue by k4leung4** [sigstore/helm-charts#192]
**Question**

Curious what people are using to generate the JSON schema, as I'm not sure what I'm doing is the best way.
I've been using 
- YAML to JSON from https://jsonformatter.org/yaml-to-json, with the values.yaml
and
- JSON Schema from https://www.jsonschema.net/app/schemas/0, with the output above to generate the schema

Do other people have better tooling for this?

**Comment by priyawadhwa** [comment]
I don't think this is better, but I've been using this (deprecated) helm plugin -- https://github.com/karuppiah7890/helm-schema-gen

**Comment by sabre1041** [comment]
Funny that you should mention it. Like. @priyawadhwa mentioned, I have gone down a similar path to what you have mentioned as well @k4leung4 . I will note that in the past, I have used some tools, such as [Readme Generator For Helm](https://github.com/bitnami-labs/readme-generator-for-helm), to assist in generating.

However, you bring up a good point which I'll being up on the Helm side
